bin/builder.rs

```diff
@@ -50,19 +66,19 @@ async fn main() -> eyre::Result<()> {
 
     select! {
         _ = submit_jh => {
-            tracing::info!("submit finished");
+            tracing::info!("submit is cooked");
         },
         _ = metrics_jh => {
-            tracing::info!("metrics finished");
+            tracing::info!("metrics is cooked");
         },
         _ = build_jh => {
-            tracing::info!("build finished");
+            tracing::info!("build is cooked");
         }
         _ = server => {
-            tracing::info!("server finished");
+            tracing::info!("server is cooked");
         }
         _ = authenticator_jh => {
-            tracing::info!("authenticator finished");
+            tracing::info!("authenticator is cooked");
```

Comment:
"is cooked" is cute but in a troubleshooting situation its pretty unclear what this means